### PR TITLE
Shared account visualization

### PR DESCRIPTION
**Pull request by nubsthead**

### CODE REVIEW

**Review by giuliano176 — DISMISSED** [review]

File: src/view/editor/parts/editor-calendar-selector.tsx [hunk 1]
```diff
@@ -8,23 +8,27 @@ import React, { ReactElement, useCallback } from 'react';
...
 		(value) => {
 			if (value) {
+				const root = getRoot(value.id);
+
 				const calResource = {
 					id: value.id,
 					name: value.name,
 					color: value.color,
-					owner: value.owner
+					owner: value.owner ?? (root as LinkFolder)?.owner
```
Comment (gnekoz) [inline src/view/editor/parts/editor-calendar-selector.tsx:31]: Is this cast safe? Maybe a type guard could guarantee  the correct type

File: src/view/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -54,37 +49,51 @@ const SidebarComponent: FC<SidebarComponentProps> = ({
...
+	foldersAccordionItems?.length > 0
+		? [
+				{
+					...(foldersAccordionItems?.[0] ?? {}),
+					children: [
+						...(foldersAccordionItems?.[0]?.children ?? []),
+						{
+							id: 'find_shares',
+							// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+							// @ts-ignore
```
Comment (gnekoz) [inline src/view/sidebar/sidebar.tsx:72]: Why the ts-ignore? If it is not avoidable a comment should be added

Comment (nubsthead) [reply src/view/sidebar/sidebar.tsx:72]: To be completely honest it was already there, I tried to remove it but it was complaining about that key which should not exist in the Folder type (and it is right) but at the same time I guess it is used somewhere in the sidebarAccordionMui but I am not sure so I don't know what is the real reason for it to be there, that is why I didn't add anything there

File: src/view/editor/parts/editor-calendar-selector.tsx (hunk 1 above)
Comment (nubsthead) [reply src/view/editor/parts/editor-calendar-selector.tsx:31]: What is your suggestion? I am forced to use this cast type in many different places as Folder and LinkFolder have different keys and they are used basically everywhere together. This is giving me headache in many places so every help is really appreciated

**Review by gnekoz — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

File: src/view/editor/parts/editor-calendar-selector.tsx (hunk 1 above)
Comment (gnekoz) [reply src/view/editor/parts/editor-calendar-selector.tsx:31]: Nevermind, the solution will turn to be too complicated

**Pull request merged**